### ISSUE TITLE
SNOW-963710: Inconsistent argument ordering in stage commands

### ISSUE THREAD

**Issue by sfc-gh-bdufour** [Snowflake-Labs/snowcli#526]
### SnowCLI version

@main

### Python version

3.8

### Platform

macOS-13.6.1-arm64-arm-64bit

### What happened

Stage commands are inconsistent w.r.t. whether they take a stage name or a path first:

```
$ snow object stage put --help

 Usage: snow object stage put [OPTIONS] PATH STAGE_NAME

$ snow object stage get --help

 Usage: snow object stage get [OPTIONS] STAGE_NAME [PATH]

$ snow object stage remove --help

 Usage: snow object stage remove [OPTIONS] STAGE_NAME FILE_NAME
```

As a user, it would be easier to have a consistent order, I end up having to look up each command before I use it.

**Comment by sfc-gh-astus** [comment]
We plan to change put and get to cp and arguments will be obvious

**Comment by sfc-gh-turbaszek** [comment]
Solved by #570
